finetuner/__init__.py

```diff
@@ -167,10 +168,16 @@ def fit(
     :param output_dim: The expected output dimension as `int`.
         If set, will attach a projection head.
     :param cpu: Whether to use the CPU. If set to `False` a GPU will be used.
```

Comment:
can you add here that this arg will be deprecated, and you should use `device` instead?